cache/lru/lru.go

```diff
@@ -0,0 +1,62 @@
+package lru
+
+import "container/list"
```

Comment:
Could you instead use the list implementation [here](https://github.com/TheAlgorithms/Go/blob/eaa2be2ffe9295b9eb996ec49af6e0e2e6381595/structure/linkedlist/doubly.go#L18)?

Reply:
I try to use it but it missing a lot of necessary function like `Remove` or `Move`

Reply:
Perfect opportunity for you to add that if you're interested 😄 If not let me know.

Reply:
@tjgurwara99 I Fixed it please review